test.js

```diff
@@ -13,6 +13,22 @@ describe('removes illegal characters', () => {
    });
 });
 
+describe('removes illegal hyphen placement',() => {
```

Comment:
This is trivial but please add a space after comma.

```suggestion
describe('removes illegal hyphen placement', () => {
```